### DISCUSSION TITLE
Is StrongSwan supported on QNX platform

### DISCUSSION THREAD

**Discussion by madhu542** [strongswan/strongswan#1973] [Q&A]
Hi Everyone,

Would like use StrongSwan 5.3.3 in one of our system on QNX. Please let me know whether it is supported on QNX Platform?

From the StrongSwan official site I could see the below portability platforms but QNX is not mentioned in that (attached). Could you please let me know whether it supported on QNX. Your help is much appreciated.

![StrongSwan_portability](https://github.com/strongswan/strongswan/assets/67962586/ce7463a2-ea03-464a-9cdf-fc0e8f0aaa85)

Thanks,

**Comment by tobiasbrunner** [comment]
QNX might support the [PF_KEY](https://www.qnx.com/developers/docs/7.1/#com.qnx.doc.neutrino.lib_ref/topic/i/ipsec_proto.html) and [PF_ROUTE](https://www.qnx.com/developers/docs/7.1/#com.qnx.doc.neutrino.lib_ref/topic/r/route_proto.html) interfaces that  FreeBSD and macOS use. But no idea whether our plugins are compatible or if there are other issues on that platform.
